redhat-cop/pelorus
# Pre commit hook

## Describe the behavior changes introduced in this PR

## Linked Issues?

## Testing Instructions

1. copy pre-commit script to .git/hooks and make a change to a chart.
2. Back down the chart version one increment, and directly run `_test/pre-commit` to see the changes it makes.
3. Back down the chart version two increments and directly run `_test/pre-commit` and see that it will continue to increment the chart version until the tests pass

@redhat-cop/mdt
